ftw/book/content/book.py

```diff
@@ -84,10 +84,10 @@
 
                 widget=atapi.BooleanWidget(
                     label=_(u'book_label_use_index',
-                            default=u'Embedd subject index')),
+                            default=u'Embedd subject index'),
                 description=_(u'book_help_use_index',
```

Comment:
Could you fix the indenting? I think should be indented +4 spaces.

Reply:
👍